### ISSUE TITLE
Underscore method prefix is not sniffed

### ISSUE THREAD

**Issue by mCzolko** [squizlabs/PHP_CodeSniffer#690]
In the [documentation](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#43-methods) for method declaration is this rule:

> Method names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.  

I'm having specified MethodDeclaration in my phpcs.xml
```
<rule ref="PSR2.Methods.MethodDeclaration"/>
```

But it doesn't check for the underscore on the beginning of method name.

**Comment by gsherwood** [comment]
The first thing the sniff does is check method names (not that these are methods, not global functions): https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PSR2/Sniffs/Methods/MethodDeclarationSniff.php#L65

Here is my test code:
```php
<?php

class Test {
    private function _one {
    }
    public function _two {
    }
    public function __one {
    }
}
```

Here is my output when running this single sniff:
```
$ php phpcs temp.php --standard=PSR2 --sniffs=PSR2.Methods.MethodDeclaration

FILE: temp.php
---------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
---------------------------------------------------------------------------------------------------
 4 | WARNING | Method name "_one" should not be prefixed with an underscore to indicate visibility
 6 | WARNING | Method name "_two" should not be prefixed with an underscore to indicate visibility
---------------------------------------------------------------------------------------------------
```

All looks to be working fine.

Can you supply some sample code that breaks the sniff for me?

**Comment by mCzolko** [comment]
I'll get back to you tomorrow

**Comment by mCzolko** [comment]
Oh, I'm so sorry it's an warning not error...
